select/internal/select.ts

```diff
@@ -642,6 +642,14 @@ export abstract class Select extends selectBaseClass {
     return hasSelectedOptionChanged;
   }
 
+  /**
+   * Closes the select if the inner menu closed
+   */
+  private handleClosed(e: Event) {
```

Comment:
I think Elliott [added this](https://github.com/material-components/material-web/blob/ec0a8ebf6ec973282b9a382a1f2fa347b5c82975/select/internal/select.ts#L683-L686) in https://github.com/material-components/material-web/commit/78e7c1742f04437eee3f804cc73de9d3ffaaf1bb right?

Reply:
aha so he did, 2 days after i opened this. didn't see that, my mistake.

this is just a tests PR in that case, he already pushed the same fix.